astrokat/simulate.py

```diff
@@ -16,6 +16,8 @@
 global simobserver
 simobserver = ephem.Observer()
 
+location = "ref, -30:42:39.8, 21:26:38.0, 1035.0, 0.0, , , 1.15"
```

Comment:
This is also in the `observatory` module.  Can we limit it to a single place?
https://github.com/ska-sa/astrokat/blob/203e5f55797212029cdc87bbbee86f71764f6aa2/astrokat/observatory.py#L41

Reply:
@bngcebetsha Please update `observatory.py` to pull in the location from `simulate.py` so that we only have 1 copy.

I suggest we change this line in `simulate.py` to better indicate it is a constant:
```suggestion
MEERKAT_REFERENCE_LOCATION = "ref, -30:42:39.8, 21:26:38.0, 1035.0, 0.0, , , 1.15"
```

Then in `observatory.py` we can import it and then do something like:
```python
 _ref_location = MEERKAT_REFERENCE_LOCATION
```